source/extensions/filters/network/rbac/rbac_filter.cc

```diff
@@ -39,15 +39,15 @@ Network::FilterStatus RoleBasedAccessControlFilter::onData(Buffer::Instance&, bo
                 : "none",
             callbacks_->connection().streamInfo().dynamicMetadata().DebugString());
 
-  std::string effective_policy_id;
+  std::string log_policy_id = "none";
```

Comment:
Given details are used by many filters, can you make sure it has rbac in it?  rbac_rejected_default? rbac_rejected_no_match?

Reply:
yeah, added the `responseDetail()` to prefix the response detail with `rbac_access_denied_matched_policy_`